GOVERNANCE.md

```diff
@@ -0,0 +1,133 @@
+# Rook Governance
+
+This document defines governance policies for the Rook project.
+
+## Roles
+
+Rook uses a two-tiered system of maintainer roles:
+
+* Senior Maintainers
```

Comment:
shall senior maintainers own the sign-off process for releases?

Reply:
As you suggested in Slack earlier, I'd be in support of the following release approval process:
* Major releases: 100% of maintainers, both senior and standard, approve of the release.
* Minor releases: 100% of senior maintainers approve of the release.  Judgement call (good faith) if a senior maintainer is on vacation and an urgent hotfix needs to be released.

I will follow up on this release criteria update in a separate PR.